libraries/AP_HAL_Linux/UARTDriver.cpp

```diff
@@ -24,7 +24,6 @@
 #include "UARTDevice.h"
 #include "UDPDevice.h"
 
-#include <GCS_MAVLink/GCS.h>
```

Comment:
this one is needed, and is breaking the CI build. It's where we get HAL_GCS_ENABLED which is used on the line below it.